packages/runtime-tools-dev-ui-webapp/src/api/RuntimeToolsDevUIEnvelopeApi.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { CustomLabels } from "./CustomLabels";
+import { DiagramPreviewSize } from "@kie-tools/runtime-tools-enveloped-components/dist/workflowDetails/api";
+
+export interface RuntimeToolsDevUIEnvelopeApi {
+  runtimeToolsDevUI_initRequest(association: Association, initArgs: RuntimeToolsDevUIInitArgs): Promise<void>;
+}
+
+export interface Association {
+  origin: string;
+  envelopeServerId: string;
+}
+
+export interface User {
+  id: string;
+  groups: string[];
+}
+
+export interface RuntimeToolsDevUIInitArgs {
```

Comment:
Some of there parameters can be removed as we are only dealing with workflows and not processes here: `users`, `trustyServiceUrl`, `customLabels`, `isTracingEnabled`. All references to those can be removed as well in other files, for example the "Audit investigation" menu item. 

In case of the custom labels, we can use "Workflow", "Workflows", "workflow" and "workflows" instead of this parameter fields, where it is called. 

Probably the `Auth` component added to `runtime-tools-components` might not be necessary anymore as well.